test/Response/SapiStreamEmitterTest.php

```diff
@@ -209,7 +209,7 @@ function ($bufferLength) use (& $peakBufferLength) {
             ->withBody($stream->reveal());
 
         ob_start();
-        $this->emitter->emit($response, $maxBufferLength);
+        $this->emitter->emit($response, null, $maxBufferLength);
```

Comment:
Right here is a good indication of why the order must be swapped; any change to how existing tests are written means breakage. If you swap the order, these tests will not need to change.